packages/drivers/odsp-driver/src/fetcher.ts

```diff
@@ -0,0 +1,92 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import assert from "assert";
+import { ITelemetryLogger } from "@fluidframework/common-definitions";
+import { IsoBuffer } from "@fluidframework/common-utils";
+import { OdspErrorType } from "./odspError";
+import { fetchAndParseAsBufferHelper, fetchAndParseAsJSONHelper, IOdspResponse } from "./odspUtils";
+
+export class Fetcher {
```

Comment:
I'd suggest putting comment describing this class, and also look for more descriptive name.

Reply:
Renamed and added comments